service/compose.yaml

```
@@ -0,0 +1,222 @@
+
```

YAMLlint (1.38.0)

[error] too many blank lines (1 > 0)

(empty-lines)